client/client-test/Languages.spec.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import Languages from '../components/Languages.jsx';
+import { configure, shallow } from 'enzyme';
+import Adapter from 'enzyme-adapter-react-16';
+ 
+configure({ adapter: new Adapter() });
+ 
+describe('Renders Components', () => {
+  let component;
+
+  beforeEach( () => {
+    component = shallow(<Languages languages={['English', 'Spanish']}/>)
+  });
+
+  test("should render initial layout", () => {
+    const component = shallow(<Languages languages={['English', 'Spanish']} />);
+    expect(component.getElements()).toMatchSnapshot();
+  });
+
```

Comment:
Nice!  So you're using your snaps to test against a shallow render of your component.  Interesting.  I was looking to see if you needed any user input testing but it looks like you're just rendering data.

Reply:
Thanks, I haven't tested mobile. However, I am able to make it stackable meaning the contents will adjust according to the user's screen width.